clients/csrf/csrf_token_manager_test.go

```diff
@@ -108,6 +109,19 @@ var _ = Describe("DefaultCsrfTokenUpdater", func() {
 				UpdateCookiesIfNeeded(cookies, request)
 				Expect(cookies).To(Equal(request.Cookies()))
 			})
+			It("should not get unauthorized", func() {
+				transport, cookies, request := createTransport(), createValidCookies(), createRequest(http.MethodPost)
+				transport.Cookies.Cookies = cookies
+				UpdateCookiesIfNeeded(transport.Cookies.Cookies, request)
```

Comment:
Why do you need to invoke this method here? It is invoked in `csrfTokenManager.updateToken()`